orunmila.go

```diff
@@ -330,13 +325,10 @@ func describeSubcmd(args []string) {
 	check(err)
 }
 
-func infoSubcmd(args []string) {
-	flag := flag.NewFlagSet("info", flag.ContinueOnError)
-
-	flag.Parse(args)
-
-	// TODO print an error message
-	// if db does not exists
+// parse args of the info subcommand and exec it
+func infoSubcmd() {
+	// parse global args instead
+	flag.Parse()
```

Comment:
why parse the global args ?

Reply:
removed, wasn't necessary as you noticed